meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -702,8 +705,10 @@ download_keys() {
 }
 
 get_signing_keys() {
-    wget -q https://raw.githubusercontent.com/3mdeb/3mdeb-secpack/master/$PLATFORM_SIGN_KEY -O - | gpg --import -
+    echo "Getting platform specific GPG key... "
```

Comment:
```suggestion
    echo -n "Getting platform specific GPG key... "
```

Reply:
Added in https://github.com/Dasharo/meta-dts/pull/76/commits/dafd803a5b67db2049ff76ea7a746d62caeb4acc